packages/components/src/components/modal/modal.tsx

```diff
@@ -274,18 +276,20 @@ export class Modal {
               <h2 class="modal__heading" part="heading">
                 {this.heading}
               </h2>
-              <button
-                ref={(el) => (this.closeButton = el)}
-                class="modal__close-button"
-                part="close-button"
-                onClick={() => this.emitBeforeClose('CLOSE_BUTTON')}
-                aria-label={this.closeButtonLabel}
-                title={this.closeButtonTitle}
-              >
-                <slot name="close-icon">
-                  <scale-icon-action-circle-close decorative />
-                </slot>
-              </button>
+              {this.displayCloseButton && (
```

Comment:
For some reason the focus trap breaks when the button is not there… let's talk about this 🤓

Reply:
Hi, i tested again in our `modal.html`, and I also logged the focusable elements. I didnt encounter the issue though. 

https://user-images.githubusercontent.com/23239723/189601625-eace7edb-07f9-4590-825d-e758e400ce98.mov

Reply:
I only had the issue in Storybook, but now it's also working there (just tested locally) 🎉 
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/851105/189631806-8d3f9bfd-213b-43cc-82e6-90c8ae15376d.png">